StoreTestWPF.ViewModel/Utils/RelayCommandWithSender.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System;
+using System.Windows.Input;
+
+namespace StoreTestWPF.ViewModel.Utils
+{
+    public class RelayCommandWithSender<T> : ICommand where T : class
```

Comment:
Please do not use concrete names when type assumes to be abstract. Try to give more abstract name.

Reply:
Fixed